HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,302 @@
+
+## GitHub issues
+The Hedera projects use GitHub issues today.
+Together, the projects contain more than 1,000 open issues that contain information that should be preserved since they
+contain great contributions from the community. Based on that, we propose to migrate the issues to the newly
+created repositories.
+
+## GitHub Actions 
+Next to the issues, each of the repositories uses GitHub Actions for continuous integration.
+While the GitHub Action workflow definitions are stored as part of the repository, we use custom action runners on
+hardware in the Google Cloud.
+The default GitHub action runners can not be used due to the complexity of `PROJECT`, especially for the services sub-project.
+To continue the development of `PROJECT` based on best practices and continuous integration, we propose to set up
+clones of the current hardware and action runners.
+The Hedera Hashgraph projects define teams with specific authorities.
+Those configurations should be migrated, and maintainers and committers for each sub-project should be defined.
+The complete list of GitHub users that should be added as maintainers or committers to the
+sub-projects will be submitted later.
+
+## Documentation
+The Hedera project provides a repository for the documentation of the Hedera network, including its deployment and
+utilization of ever-evolving services.
+That repository can be found at https://github.com/hashgraph/hedera-docs.
+The content of the repository uses gitbook to create complete documentation that is currently hosted
+at https://docs.hedera.com/hedera.
+We propose to create a neutral hosted version of the documentation at https://docs.PROJECT.com, removing or modifying
+existing content that is Hedera specific.
+We will work on that directly after the setup to remove all those parts from the transited repository.
+
+## DCO
+We suggest setting up a DCO process by LFX for all `PROJECT` repositories.
+The DCO documents will be based on the LF templates.
+We will provide the final DCO documents separately.
+
+## HIP
+The https://github.com/hashgraph/hedera-improvement-proposal repository contains the Hedera improvement proposals (HIP)
+of Hedera.
+For `PROJECT` we need improvement proposals and a process for them, too.
+Here, the HIP repo should be migrated to https://github.com/PROJECT/PROJECT-improvement-proposal.
+Today a frontend showing all HIPs is hosted at https://hips.hedera.com/.
+We propose to host a clone of that at https://NAME.PROJECT.com. 
+
+## Build
+As you can see in this chapter, `PROJECT` contains many sub-projects that need specific build steps and infrastructure.
+Therefore, it is important to invest in `PROJECT`’s build, release, and deployment.
+The goal must be to have a vendor-neutral build and release process led by LF.
+We propose to nominate a release manager by LF who is involved in the project’s transition as soon as possible.
```

Comment:
"led by LF" -- the LF does not provide the build and release process. This is something that needs to be provided by the project.

"nominate a release manager by LF" - what does the "by LF" mean here? This would be a release manager that is part of the project, not the LF.

Reply:
Tracy, we are in the process of hiring a release engineer who would build and coordinate the release process and "donating" them to our project either as an LF employee or contractor. I have discussed this on a couple of occasions with Daniela and Hart. The only open issue I'm waiting to hear back on is the most effective employment relationship of the resource; LF employee vs LF contractor vs. our employee contracted to the project? I hope that answers your question?